[DOCS] Adds the 8.8.2 release notes (#160211)
elastic/kibana: docs/CHANGELOG.asciidoc

* <<release-notes-8.0.0-alpha1>>

--
[[release-notes-8.8.2]]
== {kib} 8.8.2

> coming::[8.8.2]

Reviewer: ```suggestion

```

Review the following information about the {kib} 8.8.2 release.

[float]
[[fixes-v8.8.2]]
=== Bug Fixes

APM::
* Circuit breaker and performance improvements for service map {kibana-pull}159883[#159883]

> * Don't use service transaction metrics on transaction detail page {kibana-pull}159085[#159085]

Reviewer: ```suggestion
* Fixes the latency graph displaying all service transactions, rather than the selected one, on the transaction detail page {kibana-pull}159085[#159085]
```

Dashboard::
* Fixes styling of top nav bar {kibana-pull}159754[#159754]
* Fixes alias redirect and update error handling {kibana-pull}159742[#159742]
* Fixes time range regression {kibana-pull}159337[#159337]

Elastic Security::
For the Elastic Security 8.8.2 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].

Enterprise Search::
For the Elastic Enterprise Search 8.8.2 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].

Fleet::
* Fixes usage of AsyncLocalStorage for audit log {kibana-pull}159807[#159807]
* Fixing issue of returning output API key {kibana-pull}159179[#159179]

Logs::

> * Fixes log categorization UI failure due to infinite loop {kibana-pull}159090[#159090]

Reviewer: ```suggestion
* Fixes log categorization UI failure due to an infinity loop {kibana-pull}159090[#159090]
```

Machine Learning::
* Hiding pattern analysis button for non-time series data {kibana-pull}160051[#160051]

> * Fixes not blocking forced downgrades/installation if cannot delete indices {kibana-pull}159814[#159814]

Reviewer: ```suggestion
* Fixes blocking forced downgrades/installation if indices can't be deleted {kibana-pull}159814[#159814]
```

Maps::
* Fixes layer group loading indicator always on when group has non-visible layer {kibana-pull}159517[#159517]

> * Fixes geo line source not loaded unless maps application is opened {kibana-pull}159432[#159432]

Reviewer: ```suggestion
* Fixes geo line source not loading unless the Maps application is open {kibana-pull}159432[#159432]
```

> * Fixes map orphans sources on layer deletion {kibana-pull}159067[#159067]

Reviewer: ```suggestion
* Fixes Maps orphan sources on layer deletion {kibana-pull}159067[#159067]
```

Monitoring::
* Permanently hide the telemetry notice on dismissal {kibana-pull}159893[#159893]

Observability::
* Handle buildEsQuery error (such as leading wildcard) in status change {kibana-pull}159891[#159891]

Platform::
* Fixes global search crash on missing tag {kibana-pull}159196[#159196]
* Fixes a regression where the "saved_object_resolve" audit action was not being logged per object {kibana-pull}160014[#160014]

Uptime::
* Ensures that users can configure custom `Content-Type` headers for HTTP monitors in the Synthetics app {kibana-pull}159737[#159737]

> * Fixes an issue where alerting on Synthetics monitors could sometimes become delayed {kibana-pull}159511[#159511]

Reviewer: ```suggestion
* Fixes an issue where alerting on Synthetics monitors was sometimes delayed {kibana-pull}159511[#159511]
```
